calico/felix/fetcd.py

```diff
@@ -151,6 +156,7 @@ def _on_actor_started(self):
         self._resync_greenlet.start()
         self.status_reporter.start()
         self._status_reporting_greenlet.start()
+        self._usage_count_greenlet.start()
```

Comment:
I just thought of a useful improvement: if you moved the call to `start()` into `_on_status_from_driver()` in the `if status == STATUS_IN_SYNC and not self._been_in_sync:` conditional block, then we would wait to report the status unitl after the datamodel was in sync (i.e. we know about all the hosts).  That'd make the `len(self.ipv4_by_hostname)` reliable.